### ISSUE TITLE
A11y: Lose focus (and flow) navigating between Variant options with a Keyboard

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5271]
Create a variant and then in the Quickview, Tab to the variant option (radio buttons). You should be able to select the next one with an arrow key, which works once and then you have to Tab your way back to select another.

[https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html|https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html|smart-link] 

Variant selection works correctly in the product page. You can simply navigate back and forth among the options using arrow keys.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CMS-476**



[created from CMS-476, comment 506644]

**Comment by timo-haas-gh** [comment]
@Phil23 sorry for the disturbance but i currently do not see any improvements on this. I tested it in my local enviroment with SW 6.6.10.2 and the newest CMS-Extensions Plugin. The issue is also still present in the shopware demo. Do you know anything about that?

**Comment by timo-haas-gh** [comment]
@marcelbrode  hey sorry for the disturbance do you have any information on this issue? It is closed but i currently do not see any improvements on this. I tested it in my local enviroment with SW 6.6.10.2 and the newest CMS-Extensions Plugin (4.7). The issue is also still present in the shopware demo.

**Comment by marcelbrode** [comment]
@timo-haas-gh Sorry for coming back to you this late. Unfortunately, you mentioned me right before getting sick.

The change targeting this ticket is https://github.com/shopware/SwagCmsExtensions/pull/18 . Looking at the change itself, I could imagine that this would be noticable in SW 6.7, since it's about faulty twig block inheritance, but I do not know for sure.

@MartinKrzykawski Do you know more?

**Comment by MartinKrzykawski** [comment]
Hi @timo-haas-gh , thanks for your question!
I just tested it with SW 6.6.10.2 and CMS-Extensions 4.7, and it’s working as expected.
Do you have the `ACCESSIBILITY_TWEAKS` feature flag enabled? It’s required to use the a11y improvements already in SW 6.6.

The main improvement is that you can now navigate and select each available variant option using tabs and keyboard arrows — tabs for switching between option rows and arrows for moving within a row.

Please note: in the quickview modal, the context is lost after selecting an option because the quickview reloads. However, despite that, all options should still be accessible via tabs and arrows.

**Comment by timo-haas-gh** [comment]
@marcelbrode @MartinKrzykawski thank you for the explanation. Its working. Maybe add a debouncer for the submit after selecting an option? That would ensure that it is possible to navigate through the options without getting interrupted by the reload.
